events/equeue/equeue_mbed.cpp

```diff
@@ -131,7 +131,7 @@ static void equeue_sema_timeout(equeue_sema_t *s) {
 bool equeue_sema_wait(equeue_sema_t *s, int ms) {
     int signal = 0;
     Timeout timeout;
-    if (ms > 0) {
+    if (ms >= 0) {
```

Comment:
I'm worried of what can happen with a delay of `0` us. 
The timestamp is computed from the current time ([here](https://github.com/ARMmbed/mbed-os/blob/master/drivers/Ticker.cpp#L36)) and there is good chances that it will be *in the past* (or a far future!) when it will be actually set.